zebra/zebra_mpls.c

```diff
@@ -721,7 +723,9 @@ static int nhlfe_nexthop_active(zebra_nhlfe_t *nhlfe)
 		 * which will not be in the default
 		 * VRF.  So let's look in all of them
 		 */
-		ifp = if_lookup_by_index(nexthop->ifindex, VRF_UNKNOWN);
+		zns = zebra_ns_lookup(NS_DEFAULT);
+		ifp = if_lookup_by_index_per_ns(zns,
+						    nexthop->ifindex);
```

Comment:
nit: style is wrong here, please put everything in the same line.

Reply:
ok